### PR TITLE
add more resource request and limit

### PR DESCRIPTION
**Pull request by joway**

### CODE REVIEW

File: dashbase/tests/default/statefulset/table-default-p1.yaml [hunk 1]
```diff
@@ -84,6 +84,11 @@ spec:
           httpGet:
             path: /admin/healthcheck
             port: 8081
+        resources:
```
Comment (joway) [inline dashbase/tests/default/statefulset/table-default-p1.yaml]: I'm not familiar with the table's cpu usage, do you have any idea for this `request` ?

Comment (khou) [reply dashbase/tests/default/statefulset/table-default-p1.yaml]: yes, typically it should be 4 cpu units, but we should leave room for whatever random kubernetes management processes, so maybe 3.7

**Comment by khou** [comment]
> I also want to know how much memory request we should set for tables, may be depend on our `JVM OPTIONS`? @zengmin2016

usually JVM we request Xms4g and Xmx8g, but since we have so much extra memory, we can set much higher.

File: dashbase/values.yaml [hunk 2]
```diff
@@ -29,6 +29,12 @@ default:
         preStop:
           exec:
             command: ["/bin/sh", "-c", "curl -X GET http://localhost:7888/admin/quitquitquit"]
+      resources:
```
Comment (keitaf) [inline dashbase/values.yaml]: can we not set those default in the default values.yaml? If we set it, we cannot deploy the helm chart locally (or any small clusters which don't have enough resources).

Our current helm chart cannot be deployed w/o a custom values.yaml file (in order to set up Tables), so we can define those  resources in the same custom values.yaml.

**Pull request closed without merging**